dts/arm/st/f1/stm32f1.dtsi

```diff
@@ -333,7 +333,7 @@
 		};
 
 		adc1: adc@40012400 {
-			compatible = "st,stm32-adc";
+			compatible = "st,stm32-adc", "st,stm32f1-adc";
```

Comment:
The normal convention is for the most specific compatible to be listed first and then to get more generic.  So stm32f1-adc would be first, then stm32-adc.

Reply:
Thanks, I wasn't aware of this convention. Fixed now.